pkg/stub/interfaces.go

```diff
@@ -141,13 +142,19 @@ func (g *DefaultImageStreamFromFileGetter) Get(fullFilePath string) (is *imagev1
 	if err != nil {
 		return nil, err
 	}
-	switch {
-	case imagestream.Name == "jenkins":
-		imagestream = tagInPayload("2", "IMAGE_JENKINS", imagestream)
-	case imagestream.Name == "jenkins-agent-maven":
-		imagestream = tagInPayload("v4.0", "IMAGE_AGENT_MAVEN", imagestream)
-	case imagestream.Name == "jenkins-agent-nodejs":
-		imagestream = tagInPayload("v4.0", "IMAGE_AGENT_NODEJS", imagestream)
+	if g.h != nil {
+		// if we are not skipping jenkins imagestream, tag in images from payload
+		if _, ok := g.h.imagestreamFile["jenkins"]; ok {
```

Comment:
I was inclined to think that customers would not be looking to manage the agent streams since they don't come from known samples content from openshift/library

But sure perhaps that is an over reaching assumption?

Reply:
if we allow people to skip an imagestream by name i think we have to assume they might use that to skip the jenkins agent imagestreams in addition to the master imagestream.  (and conversely, just because they skip jenkins doesn't mean they want to skip the agents)

Reply:
this will go away in next push